docs/release-process/README.md

```diff
@@ -113,6 +118,38 @@ We should consider where release notes should go that are specific to the Micros
 
 It would be reasonable to have a place where release notes can be checked in: doc/go1.18-fips.html? Inside patch file descriptions with conventions to extract relevant info? Then the release-go pipeline can detect the notes, format them, and put them into e.g. the GitHub release.
 
+## Search for a running microsoft/go internal build rather than always queueing a new one.
+
+Tracked: [Release automation should search for an automatically triggered internal build before queueing a new one #593](https://github.com/microsoft/go/issues/593)
+
+## Better tracking of all active builds
+
+It can be hard to see what builds are currently running, including retry builds. Builds post a comment when they fail or succeed, but not when they start.
+
+Adding a comment when a build starts would be nice. Even better would be if the builds maintained a list of build statuses in a single comment so the dev doesn't have to correlate the start/end comments manually. This could be done by editing the release-day issue description.
+
+## Improved retry workflow
```

Comment:
I'm moving these into issues and cleaning up this section. Not sure why I started accumulating them here. 😄